docs/index.rst

```diff
@@ -43,7 +43,7 @@ The Taichi Programming Language
 
 
 .. toctree::
-   :caption: Contribution
+   :caption: Contributing
```

Comment:
nit: was this a good title? If I understand correct, `contributing` means the action of contribute, and `contribution` means the result of contribute?

Reply:
Sounds good. Let's use `Contributing` here.